static/css/components/sort-dropper.less

```diff
@@ -0,0 +1,62 @@
+.sort {
+  &-dropper {
+    display: flex;
+  }
+
+  &.btn {
+    font-size: 16px;
+    padding: 10px 50px;
+  }
+
+  &-content {
+    display: flex;
+    background: #fff;
+  }
+
+  &-content-inner {
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    margin-top: 10px;
+    padding: 5px 10px;
+    border: 1px solid;
+    border-radius: 5px;
+    border-color: #ddd;
+  }
+
+  &-content-inner a {
```

Comment:
If you change this to `a:link` your selector should win out for specificity over the default `a:link` styles! Which will mean you can remove `!important` and the styles will still be applied.

Reply:
@rebecca-shoptaw just tried that on my end and it works. 

` &-content-inner a:link {
    text-decoration: none;
    color: @black;
  }`

Reply:
Thank you for your help! Just a couple more stylistic changes on my end and ill submit again.